test/test_matmul_cuda.py

```diff
@@ -113,7 +115,8 @@ def test_cublas_addmm_alignment(self):
         self.assertEqual(out, torch.matmul(X, A.transpose(1, 0)) + B)
 
     @onlyCUDA
-    @unittest.skipIf(TEST_WITH_ROCM, "Only CUDA 11+ is supported")
+    @unittest.skipIf((torch.version.cuda is not None) and (not CUDA11OrLater), "Only CUDA 11+ is supported")
+    @skipIfRocmVersionLessThan((5, 2))
```

Comment:
Do we really need this check if we are on 5.3/5.4 already?

Reply:
We prefer to keep this check, as in our internal CI, we still test upstream PyTorch code with older ROCm versions.